File: sss.php
<?php
  include('simple_html_dom.php');
  $opts = array('http'=>array('header' => "User-Agent:MyAgent/1.0\r\n"));
  $context = stream_context_create($opts);
  $header = file_get_contents('https://streetstylestore.com/index.php?id_product=49551&controller=product',false,$context);
  $html = str_get_html($header);
  echo $html->find('h2[class=product-name]',0)->plaintext;
  echo "<br>";
  echo $html->find('span[id=our_price_display]',0)->plaintext;
  echo "<br>";
  echo "<img src='";
  echo $html->find('img',1)->src;
  echo "'>";
  // foreach($html->find('img') as $element)
  //      echo $element->src . '<br>';
?>

File: Webscapping/sss.py
import requests
from bs4 import BeautifulSoup

req = requests.get('https://streetstylestore.com/index.php?id_product=61561&controller=product')

soup = BeautifulSoup(req.content,'html.parser')
price = soup.find('span',id="our_price_display")
title = soup.find('h2', class_ = "product-name")
image = soup.find_all('img')
print(title.string)
print(price.string)
print(image[2])
